src/ActionList/Item.tsx

```diff
@@ -228,7 +240,37 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
         >
           <ItemWrapper {...wrapperProps}>
             <Selection selected={selected} />
-            {slots.leadingVisual}
+            {
+              // If it's inactive and isn't rendered in a container like ActionMenu or SelectPanel, render a leading visual with a tooltip
+              inactive && container === undefined ? (
+                <Tooltip text={inactiveText}>
```

Comment:
As the cursor is on the entire item, should the tooltip also be on the whole item or just the icon?

https://github.com/primer/react/assets/1863771/0add3f8d-b5f1-418e-b1e1-d8ce6cb985b2

Reply:
Just the icon